lib/pengine/utils.c

```diff
@@ -2079,8 +2079,14 @@ pe_fence_op(pe_node_t * node, const char *op, bool optional, const char *reason,
                 if(data->rc == RSC_DIGEST_ALL) {
                     optional = FALSE;
                     crm_notice("Unfencing %s (remote): because the definition of %s changed", node->details->uname, match->id);
-                    if (pcmk_is_set(data_set->flags, pe_flag_stdout)) {
-                        fprintf(stdout, "  notice: Unfencing %s (remote): because the definition of %s changed\n", node->details->uname, match->id);
+                    if (pcmk_is_set(data_set->flags, pe_flag_print)) {
+                        if (data_set->priv!= NULL) {
+                            pcmk__output_t *out = data_set->priv;
+                            out->info(out, "notice: Unfencing %s (remote): because the definition of %s changed",
+                                      node->details->uname, match->id);
+                        } else {
+                            fprintf(stdout, "  notice: Unfencing %s (remote): because the definition of %s changed\n", node->details->uname, match->id);
+                        }
```

Comment:
This is public API but I don't think it matters whether we print anything when priv is NULL, the output is not central to the purpose (as public API)